src/network/network_admin.h

```diff
@@ -103,7 +103,7 @@ void NetworkAdminClientInfo(const NetworkClientSocket *cs, bool new_client = fal
 void NetworkAdminClientUpdate(const NetworkClientInfo *ci);
 void NetworkAdminClientQuit(ClientID client_id);
 void NetworkAdminClientError(ClientID client_id, NetworkErrorCode error_code);
-void NetworkAdminCompanyInfo(const Company *company, bool new_company);
+void NetworkAdminCompanyInfo(const Company *company);
```

Comment:
```suggestion
void NetworkAdminCompanyNew(const Company *company);
```
Renaming this function will make it much clearer in the places where it is called what the intent is: notifying the admins that a new company was started.